app/models/application.rb

```diff
@@ -35,9 +35,16 @@ class Application < ApplicationRecord
   scope :with_targeted_delivery_funding_eligibility, -> { where(targeted_delivery_funding_eligibility: true) }
 
   validate :schedule_cohort_matches
+  # TODO: add constraints into the DB after separation
+  validates :ecf_id, presence: true, uniqueness: { case_sensitive: false }, if: -> { Feature.ecf_api_disabled? }
```

Comment:
can we remove the uniqueness index for now from both users/applications, as they are not and need to be added by the NPQ team when the data is cleaned up?